library/Rules/Uuid.php

```diff
@@ -13,19 +13,78 @@
 
 namespace Respect\Validation\Rules;
 
+use Respect\Validation\Exceptions\ComponentException;
 use function is_scalar;
 use function preg_match;
 
+/**
+ * This class validates UUIDs.
+ *
+ * Henrique Moody <henriquemoody@gmail.com>
+ * Michael Weimann <mail@michael-weimann.eu>
```

Comment:
You're missing the `@author` annotation here, also `Dick van der Heiden <d.vanderheiden@inthere.nl>` is also an author of this rule.